Automattic/wp-calypso
# Site Migration: Check the site ownership during the migration steps

## Proposed Changes
* Add new useAssertConditions to check if the current user created the new WPCOM site. 
* Add a helper to test the `useAssertConditions` 


## Testing Instructions
* Go to any step that requires a siteSlug  E.g. on the URL `/setup/site-migration/site-migration-instructions?from=[JN.site]&siteSlug=[SITE_YOU_DONT_OWNS]&siteId=[ID_FROM_SITE_YOU_DONT_OWNS]`
* Check if you were redirected to the home page.


**You can use it as values:**
siteSlug=golden-wryneck.jurassic.ninja
siteId=232057394

## Pre-merge Checklist
